doc/source/cluster/configure-manage-dashboard.md

```diff
@@ -205,6 +205,7 @@ Configure these settings using the `RAY_GRAFANA_HOST`, `RAY_PROMETHEUS_HOST`, `R
 
 * Set `RAY_GRAFANA_HOST` to an address that the head node can use to access Grafana. Head node does health checks on Grafana on the backend.
 * Set `RAY_PROMETHEUS_HOST` to an address the head node can use to access Prometheus.
+* Set `RAY_PROMETHEUS_HEADERS` for carrying additional headers when accessing Prometheus.
```

Comment:
It's weird to put this in the section "Embedding Grafana visualizations into Ray Dashboard". Instead, add a new section titled "Customize headers for requests from the Ray dashboard to Prometheus" in the same level as `#### Alternate Prometheus host location` and `#### Alternate Grafana host location`.

Reply:
Okay, this line is removed since we have a dedicated section for that now.